appveyor.bat

```diff
@@ -56,8 +56,8 @@ set RUBY_DIR=!RUBY%BIT%_DIR!
 :: Tcl
 set TCL_VER_LONG=8.6
 set TCL_VER=%TCL_VER_LONG:.=%
-set TCL32_URL=http://downloads.activestate.com/ActiveTcl/releases/8.6.6.8607/ActiveTcl-8.6.6.8607-MSWin32-x86-403667.exe
-set TCL64_URL=http://downloads.activestate.com/ActiveTcl/releases/8.6.6.8606/ActiveTcl-8.6.6.8606-MSWin32-x64-401995.exe
+set TCL32_URL=http://dl.activestate.com/org/vim-win32/Tcl-8.6.6/74955624-5471-4763-93e7-87611d2a1de1/artifact/ActiveTcl-8.6.6.8607-MSWin32-x86-403667.exe
```

Comment:
Is this URL a permanent one? Or should we download the installer and upload somewhere that can be accessed from our CI environment (e.g. in this repository)?